main/urls.py

```diff
@@ -20,6 +20,7 @@
     path('add_titles/<int:chapter_id>/',views.create_title_model_form,name='create_title_model_form'),
     path('add_question/<int:chapter_id>/',views.create_question,name='create_question'),
     path('add_chapter/<int:course_id>/',views.create_chapter,name='create_chapter'),
+    path('add_blog/',views.create_blog,name='create_blog'),
```

Comment:
Where is the option on frontend to create blog ? Can't see it anywhere

Reply:
I'll work on it

Reply:
Also, ensure that I am able to replicate the blog as closely as possible to to static one using the functionality you implemented.